jest.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     "^uuid$": "<rootDir>/node_modules/uuid/dist/index.js",
   },
   modulePaths: ["<rootDir>/src"],
+  setupFiles: ["<rootDir>/setupJest.js"],
```

Comment:
I'm not a huge fan of these one line files 😓 Can we do this instead and delete the file?
```suggestion
  setupFiles: ["whatwg-fetch"],
```

Reply:
Good one!